assets/app/scripts/controllers/pod.js

```diff
@@ -88,6 +88,26 @@ angular.module('openshiftConsole')
           }
         );
 
+
+        // outside of the above watch to avoid repeatedly generating new watches.
+        $scope.$watch('logOptions.container', function(newVal) {
+          if(!newVal) {
+            return;
+          }
+         var containerStatus = _.find(
+                                  $scope.pod.status.containerStatuses,
+                                  function(container) {
+                                  return container.name === newVal;
+                                });
+
+          var state = _.get(containerStatus, 'state');
+          var status = _.first(_.keys(state));
```

Comment:
Ah, as of `3.10` is an alias, but I see what you are saying now, `4.x` does deprecate it.  I'll update.